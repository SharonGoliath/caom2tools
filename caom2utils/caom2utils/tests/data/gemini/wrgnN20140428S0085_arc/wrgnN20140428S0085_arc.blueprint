Observation.type = OBJECT
Observation.algorithm.name = composite
# DerivedObservation.members = ('BINTABLE', ['Filename'], 1)
DerivedObservation.members = ('BINTABLE', 'Filename', 1)
Plane.dataProductType = image
Plane.calibrationLevel = 2
Chunk.position.axis.axis1.ctype = CTYPE1
Chunk.position.axis.axis2.ctype = CTYPE2
