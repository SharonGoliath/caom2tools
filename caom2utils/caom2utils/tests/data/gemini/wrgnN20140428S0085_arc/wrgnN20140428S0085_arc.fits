SIMPLE  =                    T / Fits standard                                  BITPIX  =                   16 / Bits per pixel                                 NAXIS   =                    0 / Number of axes                                 EXTEND  =                    T / File may contain extensions                    ORIGIN  = 'NOAO-IRAF FITS Image Kernel July 2003' / FITS file originator        DATE    = '2020-07-30T15:23:06' / Date FITS file was generated                  IRAF-TLM= '2020-07-30T15:23:55' / Time of last modification                     COMMENT   FITS (Flexible Image Transport System) format is defined in 'AstronomyCOMMENT   and Astrophysics', volume 376, page 359; bibcode: 2001A&A...376..359H INSTRUME= 'NIFS    '           / Instrument used to acquire data.               OBJECT  = 'Xe,Ar   '           / Object Name                                    OBSTYPE = 'ARC     '           / Observation type                               OBSCLASS= 'partnerCal'         / Observe class                                  GEMPRGID= 'GN-2014A-Q-85'      / Gemini Science Program ID                      OBSID   = 'GN-2014A-Q-85-12'   / Observation ID / Data label                    DATALAB = 'GN-2014A-Q-85-12-001-ARC' / Datalabel                                OBSERVER= 'Roth    '           / Observer                                       OBSERVAT= 'Gemini-North'       / Observatory                                    TELESCOP= 'Gemini-North'       / Telescope                                      PARALLAX=                   0. / Parallax of Target                             RADVEL  =                   0. / Heliocentric Radial Velocity                   EPOCH   =     2014.32156286926 / Epoch for Target coordinates                   EQUINOX =                   0. / Equinox for Target coordinates                 TRKEQUIN=                2000. / Tracking equinox                               SSA     = 'Pakzad  '           / SSA                                            RA      =         229.12653518 / RA of Target                                   DEC     =         -15.58164286 / Declination of Target                          ELEVATIO=     54.3740513888889 / Current Elevation                              AZIMUTH =     187.036002777778 / Current Azimuth                                CRPA    =    -95.8894017373411 / Current Cass Rotator Position Angle            HA      = '+00:16:51.05'       / Telescope hour angle                           LT      = '01:29:47.1'         / Local time at start of observation             TRKFRAME= 'FK5     '           / Tracking co-ordinate                           DECTRACK=      0.0010925314052 / Differential tracking rate Dec                 TRKEPOCH=         56775.479765 / Differential tracking reference epoch          RATRACK =    -0.00018777242254 / Differential tracking rate RA                  FRAME   = 'APPT    '           / Target coordinate system                       PMDEC   =                   0. / Proper Motion in Declination                   PMRA    =                   0. / Proper Motion in RA                            WAVELENG=               22000. / Effective Target Wavelength                    RAWIQ   = '70-percentile'      / Raw Image Quality                              RAWCC   = '80-percentile'      / Raw Cloud Cover                                RAWWV   = 'UNKNOWN '           / Raw Water Vapour/Transparency                  RAWBG   = '20-percentile'      / Raw Background                                 RAWPIREQ= 'NO      '           / PI Requirements Met                            RAWGEMQA= 'USABLE  '           / Gemini Quality Assessment                      CGUIDMOD= 'Basic   '           / Driving mode for carousel                      UT      = '11:29:47.6'         / Beginning of Observation (UT)                  M2BAFFLE= 'NEAR IR '           / Position of M2 baffle                          M2CENBAF= 'OPEN    '           / Position of M2 central hole baffle             ST      = '15:33:21.9'         / Sidereal time at the start of the exposure     XOFFSET =   -0.215043755381406 / Telescope offset in x in arcsec                YOFFSET =    0.231947194757898 / Telescope offset in y in arcsec                POFFSET =    -0.23194719475821 / Telescope offset in p in arcsec                QOFFSET =   -0.215043755381069 / Telescope offset in q in arcsec                RAOFFSET=   -0.235664899097696 / Telescope offset in RA in arcsec               DECOFFSE=    -0.21096296646063 / Telescope offset in DEC in arcsec              RATRGOFF=                   0. / Target offset in RA in arcsec                  DECTRGOF=                   0. / Target offset in DEC in arcsec                 PA      =                   1. / Sky Position Angle at start of exposure        IAA     =                 270. / Instrument Alignment Angle                     SFRT2   =               88.578 / Science fold rotation angle (degrees)          SFTILT  =               -44.89 / Science fold tilt angle (degrees)              SFLINEAR=                   8. / Science fold linear position (mm)              AOFOLD  = 'IN      '           / AO Pick-Off Mirror Position                    PWFS1_ST= 'parked  '           / PWFS1 probe state (frozen,guiding,parked)      PWFS2_ST= 'parked  '           / PWFS2 probe state (frozen,guiding,parked)      OIWFS_ST= 'parked  '           / OIWFS probe state (frozen,guiding,parked)      AOWFS_ST= 'guiding '           / AOWFS probe state (frozen,guiding,parked)      SCIBAND =                    3 / Science Ranking Band                           REQIQ   = '70-percentile'      / Requested Image Quality                        REQCC   = '80-percentile'      / Requested Cloud Cover                          REQBG   = 'Any     '           / Requested Background                           REQWV   = 'Any     '           / Requested Water Vapour                         REQMINAM=                   1. / Requested Minimum Airmass                      REQMAXAM=                  2.1 / Requested Maximum Airmass                      GCALLAMP= 'Ar      '           / GCAL Lamp name                                 GCALFILT= 'CLEAR   '           / GCAL Filter name                               GCALDIFF= 'IR      '           / GCAL Diffuser name                             GCALSHUT= 'CLOSED  '           / GCAL Shutter (OPEN or CLOSED)                  AOARA   =         229.12653518 / RA of AOWFS guide star                         AOARV   =                   0. / AOWFS Heliocentric Radial Velocity             AOAWAVEL=                7000. / AOWFS Effective Target Wavelength              AOADEC  =         -15.58164286 / Declination of AOWFS guide star                AOAEPOCH=     2014.32156286926 / Epoch for AOWFS guide star coordinates         AOAEQUIN=                   0. / Equinox for AOWFS guide star coordinates       AOAFRAME= 'APPT    '           / AOWFS Target coordinate system                 AOAOBJEC= 'Titan   '           / Object Name for AOWFS guide star               AOAPMDEC=                   0. / AOWFS Proper Motion in Declination             AOAPMRA =                   0. / AOWFS Proper Motion in RA                      AOAPARAL=                   0. / AOWFS Parallax of Target                       AOFOCUS =               -0.048 / AOWFS Focus Offset (mm)                        HUMIDITY=                   8. / The relative humidity (fraction, 0..101).      TAMBIENT=                   2. / The ambient temp (C).                          TAMBIEN2=                 35.6 / The ambient temp (F).                          PRESSURE=            462.17248 / The atmospheric pressure (mm Hg).              PRESSUR2=               61600. / The atmospheric pressure (Pa).                 DEWPOINT=                -29.8 / The dew point (C).                             DEWPOIN2=               -21.64 / The dew point (F).                             WINDSPEE=                  1.2 / The wind speed (m/s).                          WINDSPE2=               2.6844 / The wind speed (mph).                          WINDDIRE=                 191. / The wind direction (degrees).                  WINDCOVR= 'Opened  '           / Window cover position name (open|closed)       FILTER  = 'HK_G0603'           / Filter name                                    APOFFSET=                   0. / Fopl offseti (arcseconds)                      GRATING = 'K_G5605 '           / Grating name                                   GRATWAVE=                  2.2 / Grating cetntral wavelength                    APERTURE= '3.0_Mask_G5610'     / Focal plane mask                               FLIP    = 'Out     '           / Imaging (flip) mirror                          EXPRQ   =                  30. / Requested Exposure time                        DCNAME  = 'NIFS DC '           / Nifs dc name                                   PERIOD  =                  30. / Period                                         NPERIODS=                    1 / Number of periods                              EXPMODE = 'NONDESTRUCTIVE'     / Exposure mode                                  RDTIME  =                   5. / Read time                                      COADDS  =                    1 / Number of coadds summed                        BIASPWR =                   3. / Source voltage of bias P-FET                   OBSMODE = 'IFU     '           / Observing mode configuration                   LNRS    =                    1 / Number of non-destructive read pairs           DATE-OBS= '2014-04-28'         / Observation date (UT)                          INPORT  =                    1 / Number of ISS port where NIFS was located      CTYPE1  = 'RA---TAN'           / R.A. in tangent plane projection               CRPIX1  =            14.999838 / Ref pix of axis 1                              CRVAL1  =           228.916879 / RA at Ref pix in decimal degrees               CTYPE2  = 'DEC--TAN'           / DEC. in tangent plane projection               CRPIX2  =            34.000126 / Ref pix of axis 2                              CRVAL2  =           -15.527632 / DEC at Ref pix in decimal degrees              CD1_1   =             -1.0E-06 / WCS matrix element 1 1                         CD1_2   =              1.9E-05 / WCS matrix element 1 2                         CD2_1   =             -4.7E-05 / WCS matrix element 2 1                         CD2_2   =                  -0. / WCS matrix element 2 2                         RADECSYS= 'FK5     '           / R.A./DEC. coordinate system reference          MJD_OBS =         56775.479771 / Mean Julian day of observation                 DETSIZE = '[1:2048,1:2048]'    / Dimensions of detector                         DETECTOR= 'NIFS    '           / Name of detector                               NCCDS   =                    1 / Number of detector CCDS                        NAMPS   =                    4 / Number of CCD amps                             CCDNAME = 'NIFS    '           / CCD/ARRAY name                                 CCDSIZE = '[1:2048,1:2048]'    / Dimensions of CCD                              CCDSUM  = '1 1     '           / CCD binning                                    CCDSEC  = '[1:512,1:2048]'     / CCD section                                    AMPNAME = '1       '           / Amplifier name                                 AMPSEC  = '[1:512,1:2048]'     / Amplifier section                              DETSEC  = '[1:2048,1:2048]'    / Full detector size                             AMPNAME2= '2       '           / Amplifier 2 name                               AMPSEC2 = '[1024:513,1:2048]'  / Amplifier 2 section                            AMPNAME3= '3       '           / Amplifier 3 name                               AMPSEC3 = '[1025:1536,1:2048]' / Amplifier 3 section                            AMPNAME4= '4       '           / Amplifier 4 name                               AMPSEC4 = '[2048:1537,1:2048]' / Amplifier 4 section                            EXPTIME =                  30. / Exposure time (s)                              AOFREQ  =                1000. / Altair sampling frequency during exposure (Hz) AOCOUNTS=              40.0886 / Altair GS counts at end of exposure (ADU/subapeAOSEEING=             0.596913 / Altair measured seeing at end of exposure (arcsAOWFSX  =            -0.482908 / Altair WFS X position [mm] (0.621mm/arcsec)    AOWFSY  =             0.902284 / Altair WFS Y position [mm] (0.621mm/arcsec)    AOWFSZ  =             -3.83602 / Altair WFS Z position [mm]                     AOGAIN  =               0.5087 / Altair centroid gain estimate                  AONCPAF = '/aoBase/rel/input/aoNcpaNIFSp1Ngs.dat' / Altair Non-Common Path AberrCRFOLLOW= 'yes     '           / Cass Rotator follow mode (yes|no)              AONDFILT= 'INDEF   '           / Altair ND filter (in|out)                      AOFLENS = 'OUT     '           / Altair Fields Lens (in|out)                    AOFLEXF = '/aoBase/rel/data/aoFlexNIFSp1Ngs.dat' / Altair flexure file          LGUSTAGE= 'OUT     '           / LGS U stage position                           AIRMASS =                 1.23 / Mean airmass for the observation               AMSTART =                 1.23 / Airmass at start of exposure                   AMEND   =                 1.23 / Airmass at end of exposure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     RELEASE = '2014-04-28'         / End of proprietary period YYYY-MM-DD           MDF_YSHF=                  4.1 / MDF Y shift (subtracted from y_ccd)            MDF_FILE= 'nifs$data/nifs-mdf[1]' / MDF source                                  NEXTEND =                    1 / number of extensions in file                   MODE    = 'IFU               ' / Reduction mode                                 BIASVOLT=                3.000 / Array bias voltage (V)                         PIXSCALE=              0.04300 / Pixel scale in arcsec/pixel                    NSCIEXT =                   29 / Number of science extensions                   DISPAXIS=                    1 / Dispersion axis (along columns)                PREPARE = '2020-07-30T16:21:49' / UT Time stamp for NFPREPARE                   GEM-TLM = '2020-07-30T16:23:17' / UT Last modification with GEMINI              BPMFILE = 'rgnN20140428S0169_sflat_bpm.pl' / Input bad pixel mask file          WMEF    = '2020-07-30T16:22:18' / UT Time stamp for WMEF                        GAINORIG=                  2.4 / Input gain                                     RONORIG =                  6.3 / Input read-noise                               GEMCOMB = '2020-07-30T16:22:18' / UT Time stamp for GEMCOMBINE                  ORIGXSIZ=                 2040 / Original size in X                             ORIGYSIZ=                 2080 / Original size in Y                             NSCUT   = '2020-07-30T16:22:22' / UT Time stamp for NSCUT                       NSAPPWAV= '2020-07-30T16:22:28' / UT Time stamp for NSAPPWAVE                   NSREDUCE= '2020-07-30T16:23:11' / UT Time stamp for NSREDUCE                    DARKIMAG= 'tmpdark43230_1963'  / Dark current image subtracted from raw data    FLATIMAG= 'rgnN20140428S0169_flat' / Flat field image used                      NSWAVELE= '2020-07-30T16:23:17' / UT Time stamp for NSWAVELENGTH                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                END                                                                             XTENSION= 'BINTABLE'           / binary table extension                         BITPIX  =                    8 / array data type                                NAXIS   =                    2 / number of array dimensions                     NAXIS1  =                  192 / length of dimension 1                          NAXIS2  =                    5 / length of dimension 2                          PCOUNT  =                    0 / number of group parameters                     GCOUNT  =                    1 / number of groups                               TFIELDS =                    3 / number of table fields                         EXTNAME = 'PROVENANCE'                                                          TTYPE1  = 'Filename'                                                            TFORM1  = '64A     '                                                            TTYPE2  = 'Description'                                                         TFORM2  = '64A     '                                                            TTYPE3  = 'Type (member or input)'                                              TFORM3  = '64A     '                                                            END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             N20140428S0085.fits                                             Raw arc frame.                                                  member                                                          N20140503S0161.fits                                             Raw arc frame.                                                  member                                                          N20140428S0179.fits                                             Raw dark frame.                                                 input                                                           N20140428S0180.fits                                             Raw dark frame.                                                 input                                                           rgnN20140428S0169_flat.fits                                     Processed flat field frame.                                     input                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           