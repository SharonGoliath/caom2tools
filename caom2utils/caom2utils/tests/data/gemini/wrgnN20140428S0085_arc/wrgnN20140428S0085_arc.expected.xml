<?xml version='1.0' encoding='UTF-8'?>
<caom2:Observation xmlns:caom2="http://www.opencadc.org/caom2/xml/v2.4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="caom2:DerivedObservation" caom2:id="00000000-0000-0000-80a9-e59f7bb85513">
  <caom2:collection>GEMINI</caom2:collection>
  <caom2:observationID>wrgnN20140428S0085_arc</caom2:observationID>
  <caom2:algorithm>
    <caom2:name>composite</caom2:name>
  </caom2:algorithm>
  <caom2:type>OBJECT</caom2:type>
  <caom2:intent>science</caom2:intent>
  <caom2:planes>
    <caom2:plane caom2:id="a8772402-824e-491f-8ccd-5ddb5a11b9b2">
      <caom2:productID>wrgnN20140428S0085_arc</caom2:productID>
      <caom2:dataProductType>image</caom2:dataProductType>
      <caom2:calibrationLevel>2</caom2:calibrationLevel>
      <caom2:artifacts>
        <caom2:artifact caom2:id="1c95fabd-c78b-425e-8617-e16a56c65908">
          <caom2:uri>cadc:GEMINICADC/wrgnN20140428S0085_arc.fits</caom2:uri>
          <caom2:productType>science</caom2:productType>
          <caom2:releaseType>data</caom2:releaseType>
          <caom2:contentType>application/fits</caom2:contentType>
          <caom2:contentLength>23040</caom2:contentLength>
          <caom2:contentChecksum>md5:7bd0b06e8c7686099d013f055b979587</caom2:contentChecksum>
          <caom2:parts>
            <caom2:part caom2:id="e657a3ac-f6cb-460f-a9ff-1cc5db9051dd">
              <caom2:name>0</caom2:name>
              <caom2:chunks/>
            </caom2:part>
          </caom2:parts>
        </caom2:artifact>
      </caom2:artifacts>
    </caom2:plane>
  </caom2:planes>
  <caom2:members>
    <caom2:observationURI>caom:GEMINI/N20140428S0085.fits</caom2:observationURI>
    <caom2:observationURI>caom:GEMINI/N20140428S0179.fits</caom2:observationURI>
    <caom2:observationURI>caom:GEMINI/N20140428S0180.fits</caom2:observationURI>
    <caom2:observationURI>caom:GEMINI/rgnN20140428S0169_flat.fits</caom2:observationURI>
    <caom2:observationURI>caom:GEMINI/N20140503S0161.fits</caom2:observationURI>
  </caom2:members>
</caom2:Observation>
